packages/server/src/mail/mail.module.ts

```diff
@@ -34,6 +35,7 @@ export class MailModule {}
       // Use an empty class for a mock implementation
       useValue: {
         sendUserVerificationCode: () => 'fake code',
+        sendEmail: (_emailPost: sendEmailAsync) => 'fake email',
```

Comment:
what does this do?

Reply:
I don't think I did that. But it seems to serve a purpose from the comments rn. I think we can leave it in?